core/deployment/src/main/java/io/quarkus/deployment/dev/testing/JunitTestRunner.java

```diff
@@ -42,11 +42,7 @@
 import org.junit.jupiter.api.TestTemplate;
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.platform.commons.annotation.Testable;
-import org.junit.platform.engine.FilterResult;
-import org.junit.platform.engine.TestDescriptor;
-import org.junit.platform.engine.TestExecutionResult;
-import org.junit.platform.engine.TestSource;
-import org.junit.platform.engine.UniqueId;
+import org.junit.platform.engine.*;
```

Comment:
You will need to disable the star imports in your IDE.

Reply:
Done so, thanks for the hint. What about a `.editorconfig` file in the project?

Reply:
If you can provide one that makes sense, that would be awesome.

Let's not be too strict though but this in particular could be added for sure.